misc/pkg_install.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+LIB_7="vim build-essential ccache libgc-dev graphviz libunwind7-dev pkg-config"
+LIB_8="vim build-essential ccache libgc-dev graphviz libunwind8-dev pkg-config"
+MSG="The following package will be installed [ vim | build-essential | ccache | libgc-dev | graphviz | libunwind-dev | pkg-config ]"
+MSG_VIM="Do you want to set your vim for NIT ? [y/n] "
+
+. /etc/os-release
+BOOL=0
+
+function vim_conf {
+	NIT_PATH=$PWD
+	sudo mkdir -p ~/.vim/autoload ~/.vim/bundle
+	echo ".Vim directory created [OK]"
+
+	# Check if crul is instaled or not
+	CURL=$(dpkg-query -W -f='${Status}\n' "curl")
+	if [ "$CURL" = "install ok installed" ]; then
+		curl -LSso ~/.vim/autoload/pathogen.vim https://tpo.pe/pathogen.vim
+	else
+		echo "Curl instalation ...."
+		apt-get install curl
+		curl -LSso ~/.vim/autoload/pathogen.vim https://tpo.pe/pathogen.vim
+	fi
+
+	echo "call pathogen#infect()" >> /etc/vim/vimrc
+	echo "syntax on" >> /etc/vim/vimrc
+	echo "filetype plugin indent on" >> /etc/vim/vimrc
+	echo "Pathogen installation [OK]"
+
+	# NIT SUPPORT
+	cd ~/.vim/bundle
+	ln -s $NIT_PATH/misc/vim nit
+	echo "Nit support [OK]"
+
+	# SYNTAX CHECKER
+	git clone https://github.com/scrooloose/syntastic.git
+	echo "Syntax checker [OK]"
+
+}
+
+if [ $ID = "debian" ]; then
+	if [ $VERSION_ID = "7" ]; then
+		echo $MSG
+		read enter
+		apt-get install $LIB_7
+	else
+		echo $MSG
+		read enter
+		apt-get install $LIB_8
+	fi
+	BOOL=1
+fi
+if [ $ID = "ubuntu" ]; then
+	if [ $VERSION_ID = "12.04" ]; then
+		echo $MSG
+		read enter
+		apt-get install $LIB_7
+	else
+		echo $MSG
+		read enter
+		apt-get install $LIB_8
+	fi
+	BOOL=1
+fi
+if [ $BOOL = 1 ]; then
```

Comment:
Why is there a dependency between the packages and Vim? Even if the os is not recognized (let's say it's ArchLinux) you want to configure Vim!

It looks to me that you can remove the variable `BOOL`... But if you keep it, please find a more precise name like `os_supported`.